### PR TITLE
Incorporate Readme Updates From Tech Writers

### PR DESCRIPTION
**Pull request by ndurell**
* Removed section on shopify for now.
* Removed swift 5.3-5.4 since these weren't actually supported.
* Overall it's mostly the same just some grammar and title changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,10 +76,21 @@ Klaviyo.sharedInstance.trackEvent(
     properties: propertiesDictionary
 )
 ```
+### Arguments
+
+The `track` function can be called with up to four arguments.
+
+* `eventName`: The name of the event you want to track, as a string. This argument is required to track an event.
+
+* `customerProperties`: An NSDictionary of properties that belong to the person who did the action you're tracking. If you do not include an `$email` or `$id key`, the event cannot be tracked by Klaviyo. This argument is optional but recommended.
```
[the remark this answers is not in the document]
Comment (ndurell) [reply README.md:85]: Yup!

File: README.md [hunk 2]
```diff
@@ -88,151 +103,149 @@ personInfoDictionary[klaviyo.KLPersonZipDictKey] = "02215"
...
-
+    import UserNotifications
 ...
 
-let center = UNUserNotificationCenter.current()
-center.delegate = self as? UNUserNotificationCenterDelegate
-let options: UNAuthorizationOptions = [.alert, .sound, .badge, .provisional]
+    let center = UNUserNotificationCenter.current()
+    center.delegate = self as? UNUserNotificationCenterDelegate
+    let options: UNAuthorizationOptions = [.alert, .sound, .badge, .provisional]
```
Comment (ajaysubra) [inline README.md:162]: Do we typically add a tab to code samples? I'm seeing that it's consistent across all code samples? I don't have a strong leaning one way or the other but just a nit suggestion to may be keep them all the same.
Ex:
image

**Review by ajaysubra — COMMENTED** [review]
couple minor comments else looks good.

File: README.md [hunk 3]
```diff
@@ -88,151 +103,149 @@ personInfoDictionary[klaviyo.KLPersonZipDictKey] = "02215"
...
-To be able to send push notifications, you must add a few snippets of code to your application. One to register users for push notifications, one that will send resulting push tokens to Klaviyo, and some final snippets to handle when users attempt to open your push notifications.
+### Sending push notifications
 
-Add the below code to your application wherever you would like to prompt users to register for push notifications. This is often included within `application:didFinishLaunchingWithOptions:`, but it can be placed elsewhere as well. Make sure that whenever this code is called that the Klaviyo SDK has been configured and that `setUpUserEmail:` has been called. This is so that Klaviyo can match app tokens with customers.
+1. Add the following code to your application wherever you would like to prompt users to register for push notifications. This is often included within `application:didFinishLaunchingWithOptions:`, but it can be placed elsewhere as well. When this code is called, ensure that the Klaviyo SDK has beenis configured and that `setUpUserEmail:` has beenis called. This enables Klaviyo to match app tokens with profiles in Klaviyocustomers.
```
[the remark this answers is not in the document]
Comment (ndurell) [reply README.md:154]: OMG

[the remark this answers is not in the document]
Comment (jordan-griffin) [reply README.md:154]: hahaha

[the remark this answers is not in the document]
Comment (ndurell) [reply README.md:154]: gonna change has beenis to is I think.

File: README.md [hunk 4]
```diff
@@ -88,151 +103,149 @@ personInfoDictionary[klaviyo.KLPersonZipDictKey] = "02215"
...
 ```
 
-That is all you need to do to track opens. Now once your first push notifications have been sent and been opened, you should start to see `Opened Push` metrics within your Klaviyo dashboard.
+Once your first push notifications are sent and opened, you should start to see *Opened Push* metrics within your Klaviyo dashboard.
 
-## [OPTIONAL] Foreground Push Handling
+### Foreground push handling
 
-The code below will enable push notifications to show up when you app is running:
+The following code example allows push notifications to appearshow when your app is running:
```
Comment (jordan-griffin) [inline README.md:214]: `appearshow`. I assume you want to say just `show` here

**Comment by ndurell** [comment]
> > * Removed section on shopify for now.
> > * Removed swift 5.3-5.4 since these weren't actually supported.
> > * Overall it's mostly the same just some grammar and title changes.
> 
> Also, I'm still seeing the section on shopify. Am I missing something?
> 
> > ## Integrating with Shopify's Mobile SDK

Good catch!

File: README.md (hunk 4 above)
Comment (ndurell) [reply README.md:209]: 😢 Yes

Comment (ndurell) [reply README.md:209]: Gonna re-word.

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (ndurell) [reply README.md]: Yah I think I got both.

**Review by jordan-griffin — APPROVED** [review]

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (ndurell) [reply README.md]: outdated I think maybe take a look again.

File: README.md (hunk 2 above)
Comment (ndurell) [reply README.md:157]: actually I might leave this for now. We can re-visit the indentation later.

**Pull request merged**